src/controller/DanmakuControllerDto.d.ts

```diff
@@ -4,6 +4,8 @@
 type BasicDanmakuDto = {
 	/** 非空 - KVID 视频 ID */
 	videoId: number;
+	/** 非空 - 用户 UUID */
+	UUID: string;
```

Comment:
修正变量拼写：应该改为 uuid（全部小写），下同。


```suggestion
	uuid: string;
```